package.json

```diff
@@ -1,18 +1,28 @@
 {
   "private": true,
   "scripts": {
-    "prod": "gulp --production",
-    "dev": "gulp watch"
+    "dev": "npm run development",
+    "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "watch-poll": "npm run watch -- --watch-poll",
+    "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "prod": "npm run production",
+    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
   },
   "devDependencies": {
-    "bootstrap-sass": "^3.3.7",
+    "animate.css": "^3.5.2",
+    "bootstrap": "4.0.0-alpha.2",
+    "cross-env": "^5.0",
+    "font-awesome": "^4.7.0",
     "gulp": "^3.9.1",
-    "jquery": "^3.1.0",
-    "laravel-elixir": "^6.0.0-14",
-    "laravel-elixir-vue-2": "^0.2.0",
-    "laravel-elixir-webpack-official": "^1.0.2",
+    "hint.css": "^2.3.2",
+    "list.js": "^1.5.0",
+    "jquery": "^3.2.1",
+    "jquery-tags-input": "git://github.com/xoxco/jQuery-Tags-Input#v1.3.6",
```

Comment:
I had to change this by `jQuery-Tags-Input` to make it work. Is that the case for you as well?

Reply:
Yeah, probably because my homestead instance was on Windows.